src/lib/table/table.spec.ts

```diff
@@ -120,6 +123,7 @@ describe('MatTable', () => {
       // Change filter to empty string, should match all rows
       dataSource.filter = '';
       fixture.detectChanges();
+      expect(dataSource.filteredData.length).toBe(3);
```

Comment:
Add a test that the filtered data is _correct_?

Reply:
Added a bit more to the test - good catch